packages/components/src/internal/components/pipeline/constants.ts

```diff
@@ -0,0 +1,22 @@
+import { Map } from 'immutable';
+import { Filter } from '@labkey/api';
+
+const SAMPLES_IMPORT_PIPELINE_PROVIDER = 'Samples Import';
+const SOURCES_IMPORT_PIPELINE_PROVIDER = 'Sources Import';
+
+export const PROVIDERS = [SAMPLES_IMPORT_PIPELINE_PROVIDER, SOURCES_IMPORT_PIPELINE_PROVIDER, 'General'];
```

Comment:
'Sources Import' is specific to LKSM only. For LKB, are we enabling async import for registry data?

Reply:
Oh, nope. I've made an edit where PipelineJobsListingPage now uses different filters depending on if SM or LKB is the app being used.